app/components/rails_bootstrap/badge_component.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class RailsBootstrap::BadgeComponent < ApplicationComponent
+  
+  VARIRANTS = ["primary", "secondary", "success", "danger", "warning", "light", "info", "dark"].freeze
+  PILLS = [true, false].freeze
```

Comment:
You can simply write this inline when you validate. I feel like we don't need to store this in a variable